### DISCUSSION TITLE
Site not rendered properly, when using hx- prefix in classes

### DISCUSSION THREAD

**Discussion by fmunteanu** [imfing/hextra#377] [Q&A] [answered]
Hi all,

First, thank you @imfing for creating this beautiful Hugo theme.

I have a small issue that I'm trying to figure the root cause: using the default `hx-` prefix in classes breaks the site design. See [deployed site](https://fmunteanu.github.io/hugo-test/) and related [source code](https://raw.githubusercontent.com/fmunteanu/hugo-test/main/docs/content/_index.md), after the GitHub workflow deployment.

While testing everything locally, I noticed that removing the  `hx-` prefix fixes the issues:

![image](https://github.com/imfing/hextra/assets/19806136/b9bbc659-ef0b-4878-9a85-73d53b1cf27c)

I followed the provided guidelines into [documentation](https://imfing.github.io/hextra/docs/), is there something I missed into my [hugo.yaml](https://github.com/fmunteanu/hugo-test/blob/main/docs/hugo.yaml)?

Commands I executed to generate the Hugo build:

```
$ hugo version
hugo v0.125.4+extended darwin/amd64 BuildDate=2024-04-25T13:27:26Z VendorInfo=brew
$ hugo new site docs --format=yaml
$ cd docs
$ hugo mod init github.com/fmunteanu/hugo-test/docs
$ hugo mod get -u
$ hugo new content/_index.md
```

Thank you for your help.

**Comment by fmunteanu** [comment, marked as answer]
I think I understand now why the CSS was not rendered properly, the #300 is not currently part of Hextra releases. After running this command:
```
hugo mod get -u github.com/imfing/hextra@main
```

Everything started working properly. @imfing any idea when a new release will be pushed to include the PR?

**Reply by imfing** [reply]
Hi, glad you figured that out :) 

> any idea when a new release will be pushed to include the PR?

No exact date but will be soon. Just wrapping up a few PRs.

**Reply by fmunteanu** [reply]
@imfing I have two feature requests I want to post into Issues, I already implemented them into my wiki and I think other users will find them useful. Should take you 5 minutes to implement them. 😊

**Reply by imfing** [reply]
> @imfing I have two feature requests I want to post into Issues, I already implemented them into my wiki and I think other users will find them useful. Should take you 5 minutes to implement them. 😊

please put them in new issues 🙏 thanks

**Reply by fmunteanu** [reply]
> please put them in new issues 🙏 thanks

Will do, @imfing, I'll detail the feature requests I already implemented. I also found 2 bugs which I will report also. Quick preview of the wiki, using your awesome theme:

![image](https://github.com/axivo/k3s-cluster/assets/19806136/cd01dc85-80be-4cff-a97c-cba1e55d84ba)
